apps/explorer/components/Layout/Footer.tsx

```diff
@@ -49,7 +50,9 @@ export function Footer() {
           <div className="flex-1" />
           <div className="flex-1 flex items-center justify-end gap-6">
             <CurrencyFiatSelector />
-            <ThemeRadio className="hidden md:flex" />
+            <ClientSideOnly>
```

Comment:
This will always be necessary for ThemeRadio so you can move it inside that component.

Reply:
Happy to do this but it's worth noting that it isn't always necessary for this component. We use this in Explorer's menu and NextJS isn't hydrating that version of the component because it isn't visible. You can test this out by getting the hydration warning on main and then removing the footer ThemeRadio. That particular warning will go away.

Reply:
I see what you are saying, but this requires the caller to understand the implementation details - with this logic internalized the component is safe to use however.

Reply:
The other added benefit is that you can internally set the fallback to look exactly the same rather than externally set a equivalent sized box. If someone ever changes ThemeRadio that box (and fallbacks in other apps) may fall out of sync.